sykemelding-fodselspermisjon.md

```diff
@@ -0,0 +1,61 @@
+opp til tilsvarende 6G, hvor Apparat vil betale ut resterende beløp opp til 90% av normal lønn. Det undersøkes om Apparat 
+skal forskutere hele beløpet men rutinen for dette er enda ikke helt på plass. Under følger to eksempler på hvordan 
+det vil fungere i praksis for deg som ansatt i tiden etter arbeidsgiverperioden: 
+
+> <b>Du er sykemeldt 100% for en lenger periode </b> Det vil si at NAV vil utbetale lønn tilsvarende årslønn opp til 6G, mens
+> Apparat vil betale den biten av lønna fra 6G opp til faktisk lønn </br>
+> Eks: For enkelhetsskyld sier vi at 6G vil tilsvare 40K i mnd, la oss si at du tjener 8G og at det tilsvarer 50K i mnd.
+> La oss si at du er sykemeldt utenfor arbeidsgiverperioden en hel mnd. Det vil si at NAV kommer til å utbetale 40K til deg.
+> Fra apparat sin side så vil det at du skal ha 90% utbetalt bety at du skal få utbetalt opp til 45K totalt fra Apparat og NAV.
+> Altså vil Apparat utbetale 5K for perioden, alt dette er før skatt. 
+
+> <b>Du er gradert sykemeldt opp til 40% av perioden.</b> Så si du jobber 75% i den gitte perioden. Da vil man få 100% lønn
+> for det man faktisk jobber, mens man vil få 90% på de resterende timene man ikke jobber. </br> 
+> Eks: La oss bruke de samme tallene som i forrige eksempel, at NAV vil utbetale opp til 40K før skatt og at du tjener 50K 
+> i vanlig mndslønn hos Apparat. // TODO  
+
+### Foreldrepermisjon
+Foreldrepermisjon har mye til felles med sykemeldingseksemplemene over. Vi kommer ikke til å gå i dybden på hvordan det
+vil fungere i praksis om man bare tar ut deler av foreldrepermisjon, eller splitter permisjonen veldig granulert. 
+
+Man oppfordrer selvfølgelig til en 50/50 fordeling av permisjonen i likestillingens navn, men Apparat vil utbetale opp til 90%
+av lønn for den lovtildelte foreldrepermisjonen, for den ansatte i Apparat, uansett hvordan man tildeler kvoten. 
+Dvs at om man tar mer permisjon enn det man får støtte til fra NAV, må man fremdeles ta denne ulønnet. 
+
+Det vil si at hvis ansatt er fødende mor vil det si opp til 3 + 15 + (x av 16) uker, og hvis ansatt er far/medmor
+vil det si 2 + 15 + (x av 16) uker, disse tallene kan endre seg om staten endrer reglene rundt tildeling av foreldrepermisjon. 
+Se NAV sine sider om foreldrepermisjon for oppdatert informasjon om foreldrekvoter og antall uker osv.
+
+Apparat jobber med å undersøke hvordan det vil fungere i praksis å forskuttere hele lønna i permisjonsperioden, men 
+rutinene for dette er enda ikke helt på plass, så foreløpig vil man få en to-delt utbetaling, hvor deler kommer fra NAV og 
+deler kommer fra Apparat. 
+
+> Eks: NAV dekker 6G, som er ~640.000 kr. Si at du har 8G i lønn altså ~850.000 kr i lønn. Det er 210.000 kr i differanse 
+> mellom de to tallene og 90% av differansen er 190.000 kr. Så 190.000 kr vil være den andelen av lønnen som Apparat
+> dekker. 190.000kr/12mnd  ~15.800kr i mnd + pensjon, forsikring og fp/aga? 
```

Comment:
usikker hva fp/aga står for men det er noen av godene vi får gjennom Apparat, så hvis noen har fullt ut hva det er så hadde det vært toppers :)